storage/object/reference-content/cname-record.mdx

```diff
@@ -0,0 +1,89 @@
+---
+meta:
+  title: CNAME records and DNS for Object Storage with Edge Services
+  description: Learn how to create a CNAME record for your customized Edge Services domain, and troubleshoot any potential DNS problems.
+content:
+  h1: CNAME records and DNS for Object Storage with Edge Services
+  paragraph: Learn how to create a CNAME record for your customized Edge Services domain, and troubleshoot any potential DNS problems.
+tags: object-storage edge-services cname dns
+categories: 
+  - storage
+---
+
+This document contains information to help you successfully create a CNAME record for your customized [Edge Services](/storage/object/how-to/get-started-edge-services) domain, and troubleshoot any potential DNS problems.
+
+## What is a CNAME record?
+
+A **C**anonical **Name** (CNAME) record is a type of [DNS record](/network/domains-and-dns/concepts/#dns-record). Generally, DNS records hold information for translating a domain or subdomain to an IP address, mail server or other domain/subdomain. They are crucial in directing internet traffic to the correct servers. More specifically, CNAME records map one domain name (an alias) to another (the canonical name). 
+
+A CNAME record may look like the following:
+
+| Hostname / Alias           | Destination / Canonical Name  |
+|----------------------------|-------------------------------|
+| `videos.example.com`       | `otherdomain.com`             |
+
+In this case, when a DNS server sees this record for `videos.example.com` it will know not to direct traffic to `videos.example.com`'s own IP address, but to that of `otherdomain.com`. It will find `othercomain.com`'s IP address via its [A record](/network/domains-and-dns/concepts/#resource-records).
```

Comment:
```suggestion
In this case, when a DNS server sees this record for `videos.example.com` it will know not to direct traffic to `videos.example.com`'s own IP address, but to that of `otherdomain.com`. It will find `othercomain.com`'s IP address via its [A record](/network/domains-and-dns/reference-content/understanding-dns-records/#a-record).
```